### PR TITLE
reference: add nohup usage for mydumper/loader/syncer and add new flags in mydumper

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

Fix PRM-102
Mydumper add several new flags while documents are not updated yet 

### Which version does your change affect? 

dev/v3.0/v2.1

### CODE REVIEW

**Comment by TomShawn** [comment]
@ericsyh Please fix the CI failure.

**Comment by ericsyh** [comment]
@TomShawn Thx, CI fixed.

File: dev/reference/tools/loader.md [hunk 1]
```diff
@@ -129,6 +129,14 @@ max-allowed-packet = 67108864
 
 ```bash
 ./bin/loader -c=config.toml
+```
+
+nohup 启动：
```
Comment (TomShawn) [inline dev/reference/tools/loader.md]: `nohup 启动` or `启动 nohup`?

File: dev/reference/tools/mydumper.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ Mydumper 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/dev/r
   -z, --tidb-snapshot: 设置 tidb_snapshot 用于备份
                        默认值：当前 TSO（SHOW MASTER STATUS 输出的 Position 字段)
                        此参数可设为 TSO 或有效的 datetime 时间，例如：-z "2016-10-08 16:45:26"
+  -r, --rows:          设置导出表时候拆分行数
```
Comment (TomShawn) [inline dev/reference/tools/mydumper.md]:
```suggestion
  -r, --rows:          设置导出表时候拆分的行数
```

File: dev/reference/tools/mydumper.md [hunk 3]
```diff
@@ -26,6 +26,10 @@ Mydumper 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/dev/r
...
+                       此参数将覆盖 chunk-filesize 的配置
+  -w, --where:         设置导出的过滤条件
+                       如果过滤条件包含中文字段，需要中文字段进行 utf8 encode 后使用
```
Comment (TomShawn) [inline dev/reference/tools/mydumper.md]:
```suggestion
                       如果过滤条件包含中文字段，需对中文字段进行 utf8 编码后才可使用
```

File: dev/reference/tools/mydumper.md [hunk 4]
```diff
@@ -45,6 +49,14 @@ Mydumper 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/dev/r
 ./bin/mydumper -h 127.0.0.1 -u root -P 4000
 ```
 
+nohup 启动：
```
Comment (TomShawn) [inline dev/reference/tools/mydumper.md]: `nohup 启动` or `启动 nohup`?

File: dev/reference/tools/syncer.md [hunk 5]
```diff
@@ -215,6 +215,14 @@ port = 4000
 2016/10/27 15:22:01 syncer.go:549: [info] rotate binlog to (mysql-bin.000003, 1280)
 ```
 
+nohup 启动：
```
Comment (TomShawn) [inline dev/reference/tools/syncer.md]: ditto

File: dev/reference/tools/loader.md (hunk 1 above)
Comment (ericsyh) [reply dev/reference/tools/loader.md]: nohup 启动

File: dev/reference/tools/mydumper.md (hunk 4 above)
Comment (ericsyh) [reply dev/reference/tools/mydumper.md]: nohup 启动

**Comment by TomShawn** [comment]
@ericsyh Please apply the comments to v2.1 and v3.0, and involve a technical review.

**Comment by TomShawn** [comment]
@ericsyh When the content is ready, please also update v3.1 as well, thanks~

**Comment by ericsyh** [comment]
@kennytm Kenny pls help a review~

File: dev/reference/tools/mydumper.md [hunk 6]
```diff
@@ -26,6 +26,10 @@ Mydumper 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/dev/r
   -z, --tidb-snapshot: 设置 tidb_snapshot 用于备份
                        默认值：当前 TSO（SHOW MASTER STATUS 输出的 Position 字段)
                        此参数可设为 TSO 或有效的 datetime 时间，例如：-z "2016-10-08 16:45:26"
+  -r, --rows:          设置导出表时候拆分的行数
+                       此参数将覆盖 chunk-filesize 的配置
+  -w, --where:         设置导出的过滤条件
+                       如果过滤条件包含中文字段，需对中文字段进行 utf8 编码后才可使用
```
Comment (kennytm) [inline dev/reference/tools/mydumper.md]: we should explain what is meant by "utf8 编码" and how to do it

**Comment by ericsyh** [comment]
Prefer put the introduction of nohup usage on asktug rather formal documents. So i close this PR.

**Pull request closed without merging**